tests/basic/storage_pool_test.go

```diff
@@ -5794,3 +5794,196 @@ func addNewPools(n node.Node, numPools int) error {
+			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+
+			resizeErr := waitForPoolToBeResized(expectedSize, poolUUID, isjournal)
+			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool [%s] on node [%s] expansion using auto", poolUUID, nodeDetail.Name))
+		})
+
+		// Validate if PoolIO Priority is not changed after pool Expansion
+		ioPriorityAfter, err := GetPoolIOPriorityStatus(poolUUID)
+		log.FailOnError(err, "Failed to get IO Priority for Pool with UUID [%v]", poolUUID)
+
+		log.InfoD(fmt.Sprintf("Priority Before [%s] and Priority after Pool Expansion [%s]", ioPriorityBefore, ioPriorityAfter))
+
+		opts := make(map[string]bool)
+		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+		ValidateAndDestroy(contexts, opts)
+
+		stepLog = "destroy all the applications created before test runs"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			opts := make(map[string]bool)
+			opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+			for _, ctx := range contexts {
+				TearDownContext(ctx, opts)
+			}
+		})
+	})
+
+	JustAfterEach(func() {
+		// Making sure that the nodes are not in Mainteinance mode
+		ExitNodesFromMaintenanceMode()
+		defer EndTorpedoTest()
+		AfterEachTest(contexts, testrailID, runID)
+	})
+})
+
+var _ = Describe("{VerifyPoolDeleteInvalidPoolID}", func() {
+	var testrailID = 79487
+	// Testrail Description : Verify deletion of invalid pool ids
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/55349
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("VerifyPoolDeleteInvalidPoolID",
+			"Verify deletion of invalid pool ids",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Verify deletion of invalid pool ids"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("deleteinvalidpoolid-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
+
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+
+		PoolDetail := GetPoolsDetailsOnNode(*nodeDetail)
+		log.InfoD("Pool Details [%v] Picked for Delete", PoolDetail)
+
+		// invalidPoolID is total Pools present on the node + 1
+		invalidPoolID := fmt.Sprintf("%d", len(PoolDetail)+1)
+
+		log.InfoD("Bring Node to Maintenance Mode")
+		log.FailOnError(Inst().V.EnterMaintenance(*nodeDetail), fmt.Sprintf("Failed to bring Pool [%s] to Mainteinance Mode on Node [%s]", poolUUID, nodeDetail.Name))
+
+		// Wait for some time before verifying Maintenance state
+		time.Sleep(5 * time.Minute)
```

Comment:
I dont think this wait is required as we are checking the status inside EnterMaintenance function

Reply:
I did see a behaviour where the status shows "In maintenance " but it took some time to come out of quorum , so thought of introducing a sleeptime  of 5 min ,
will observe the behaviour once again and delete the time sleep if not needed